servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestContents.java

```diff
@@ -163,24 +210,129 @@ public void verifyContentAndOperationTypesIndividually(
             .branch(branch)
             .commitMeta(CommitMeta.fromMessage("commit " + contentAndOperationType))
             .operation(contentAndOperationType.operation);
-    if (contentAndOperationType.globalOperation != null) {
-      commit.operation(contentAndOperationType.globalOperation);
-    }
-    commit.commit();
-    List<Entry> entries = getApi().getEntries().refName(branch.getName()).get().getEntries();
+    Branch committed = commit.commit();
+
     // Oh, yea - this is weird. The property ContentAndOperationType.operation.key.namespace is null
-    // (!!!)
-    // here, because somehow JUnit @MethodSource implementation re-constructs the objects returned
-    // from
-    // the source-method contentAndOperationTypes.
+    // (!!!) here, because somehow JUnit @MethodSource implementation re-constructs the objects
+    // returned from the source-method contentAndOperationTypes.
     ContentKey fixedContentKey =
         ContentKey.of(contentAndOperationType.operation.getKey().getElements());
-    List<Entry> expect =
-        contentAndOperationType.operation instanceof Put
-            ? singletonList(
-                Entry.builder().name(fixedContentKey).type(contentAndOperationType.type).build())
-            : emptyList();
-    assertThat(entries).containsExactlyInAnyOrderElementsOf(expect);
+
+    if (contentAndOperationType.operation instanceof Put) {
+      Put put = (Put) contentAndOperationType.operation;
+      assertAll(
```

Comment:
same here, also at some point this gets hard to read, so I would prefer to not use `assertAll`